_posts/2021-09-27-solidity-0.8.8-release-announcement.md

```diff
@@ -101,6 +101,31 @@ inside the constructor but you cannot use them to initialize other
 immutable variables at the point of their declaration.
 
 
+### Import Paths
+
+We added a new option `--include-path` to `solc` that hopefully makes it much easier to compile
+more complicated projects using multiple libraries. You can use it to specify
```

Comment:
Might be worth clarifying that it affects the default import callback. I.e. all CLI usage, with or without `--standard-json` but not usage via solc-js (unless someone uses the `solcjs` script to invoke the compiler).

Reply:
I considered it, but isn't that use-case a bit too specific and might confuse the "ordinary peopel"? I mean this talk about "options"  - and if you use the binary interface, there is no real place to provide these options, isn't it?

Reply:
True, but what I'm worried about is that it's not clear that it affects `--standard-json` too. Though maybe it's ok - most people probably are also not aware that with `--standard-json` imports might still get loaded from disk.